snare/tests/test_submit_data.py

```diff
@@ -65,6 +69,21 @@ def test_event_result(self):
         self.loop.run_until_complete(test())
         self.assertEqual(self.result, dict(detection={'type': 1}, sess_uuid="test_uuid"))
 
+    def test_submit_data_error(self):
+
+        async def mock_json_decode_error(data):
+            logger.error('Error submitting data: JSONDecodeError {}'.format(data))
```

Comment:
Usually, logger creates a file for errors, is this also doing the same? If so please do the cleanup.